src/xenia/base/string_util.h

```diff
@@ -77,15 +77,22 @@ inline std::string to_string(const __m128& value) {
   char buffer[128];
   float f[4];
   _mm_storeu_ps(f, value);
-  std::snprintf(buffer, sizeof(buffer), "(%F, %F, %F, %F)", f[0], f[1], f[2],
-                f[3]);
+  snprintf(buffer, sizeof(buffer), "(%F, %F, %F, %F)", f[0], f[1], f[2], f[3]);
```

Comment:
I don't think this belongs to this PR. Why did you drop `std::`?

Reply:
Clarified the drop of std:: in the commit message, copying here: 
This seems to be an issue with Microsoft declaring a snprintf macro in an included header which causes issues.
Related issues:
- https://github.com/nlohmann/json/issues/1408
- https://github.com/kodi-pvr/pvr.vuplus/issues/162